tutorials/tour/_posts/2017-02-13-type-inference.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: tutorial
+title: Type Inference
+
+disqus: true
+
+tutorial: scala-tour
+categories: tour
+num: 29
+next-page: operators
+previous-page: polymorphic-methods
+prerequisite-knowledge: unified-types, generic-classes
+---
+The Scala compiler can often infer the type of a value so you don't have to declare it explicitly. You'll often see this with variables and return types.
```

Comment:
and I'm not sure "You'll often see this with variables and return types" really adds any meaning?